build-system/server/app.js

```diff
@@ -753,6 +753,22 @@ app.post('/get-consent-no-prompt/', (req, res) => {
   res.json(body);
 });
 
+app.post('/check-consent', (req, res) => {
```

Comment:
Instead of introducing `mode` concept, we can echo back what ever key-value pairs specified in the request:

```
/check-consent?consentRequired=true&expireCache=true
```